secretmanager/secretmanager_test.go

```diff
@@ -577,6 +577,34 @@ func TestUpdateSecret(t *testing.T) {
 	}
 }
 
+func TestUpdateSecretWithAlias(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	secret := testSecret(t, tc.ProjectID)
+	defer testCleanupSecret(t, secret.Name)
+
+	var b bytes.Buffer
+	if err := updateSecretWithAlias(&b, secret.Name); err != nil {
+		t.Fatal(err)
+	}
+
+	if got, want := b.String(), "Updated secret"; !strings.Contains(got, want) {
+		t.Errorf("updateSecret: expected %q to contain %q", got, want)
```

Comment:
updateSecretWithAlias

Reply:
This would be inconsistent with the etags case, it's still a call to updatesecret